pgmpy/models/BayesianNetwork.py

```diff
@@ -1253,15 +1251,9 @@ def simulate(
         show_progress: bool
             If True, shows a progress bar when generating samples.
 
-        include_missing: bool (default: False)
-            If True, include missing values in the samples.
-
-        missing_prob: float (default: 0.1)
-            The probability that there is missing values in the samples.
-
-        missing_columns: list (default: None)
-            The list of columns where there will be missing values in the samples.
-            If None, then all columns could contain the missing values.
+        missing_prob: TabularCPD, list  (default: None)
+            The probability of missing value for the variable of TabularCPD.
```

Comment:
We should specify what the variable names in these CPDs should look like such as the variable name and the state names.